es/servicios.html

```diff
@@ -42,39 +42,127 @@ <h1 class="h-page">Servicios</h1>
         </section>
       </div>
 
-      <main role="main">
+      <main role="main">    
+        <div id="stickyMenu" class="sticky-menu">
+          <a href="#serviceWorkshop" class="sticky-menu-item sticky-menu-workshop">
+            <div class="sticky-icon icon-workshop-black"></div>
+            <h4>Talleres</h4>
+          </a>  
 
-        <section class="l-group section-wrap">
-          <h3>Talleres y Cursos</h3>
-          <button>Ver siguientes talleres</button>
-          <button>Solicitar para mi empresa</button>
-          <p>Nuestro talleres son abiertos al público (ver calendario*) y también ofrecemos versiones hechas a la medida para su empresa u organización. Abordamos la aplicación de la CNV en temas como: Comunicación Empática, el Arte de Dialogar y la Colaboración efectiva.</p>
-          <video src=""></video>
-        </section>
+          <a href="#serviceConsultancies" class="sticky-menu-item sticky-menu-consultancies">
+            <div class="sticky-icon icon-consultancies-black"></div>
+            <h4>Consultoría</h4>
+          </a> 
+
+          <a href="#serviceCoaching" class="sticky-menu-item sticky-menu-coaching">
+            <div class="sticky-icon icon-coaching-black"></div>
+            <h4>Coaching</h4>
+          </a> 
+
+          <a href="#serviceRetreats" class="sticky-menu-item sticky-menu-retreats">
+            <div class="sticky-icon icon-retreats-black"></div>
+            <h4>Retiros</h4>
+          </a>
+        </div>    
+        <section id="serviceWorkshop" class="l-group section-wrap">        
+          <div class="service-info">
+            <h3>Talleres y Cursos</h3>
+            <div class="service-content">
+              <p>Nuestro talleres son abiertos al público (ver calendario*) y también ofrecemos versiones hechas a la medida para su empresa u organización. Abordamos la aplicación de la CNV en temas como: Comunicación Empática, el Arte de Dialogar y la Colaboración efectiva.</p>
+              <div class="btn-request-content">
+                <a class="btn-invert" href="">Request for my company</a>
+              </div>              
+            </div>           
+            <video src="../video/Video IFAD.mp4" controls=""></video>
```

Comment:
que el nombre de los archivos, imagen o video, esten todo en minuscula y sin espacios.

que la etiqueta video tenga source,

`<video>
          <source src="path/" type="video/mp4">
          Your browser does not support HTML5 video.
        </video>`